lib/core/src/Cardano/DB/Sqlite.hs

```diff
@@ -362,6 +367,10 @@ instance ToText DBLog where
             fmt $ ""+||n||+" migrations were applied to the database."
         MsgMigrations (Left err) ->
             "Failed to migrate the database: " <> getMigrationErrorMessage err
+        MsgForcedRollback -> mconcat
+            [ "Some automated database migrations happened. Forced to roll back "
+            , "to genesis."
```

Comment:
**minor suggestion**
```
Rollback to the genesis was forced.
```
it is somewhat more natural to me

Reply:
Why `the` however :thinking: ?

Reply:
yeah, probably without `the`. so no `to the genesis` but `to genesis`